WcaOnRails/config/wca-states.json

```diff
@@ -0,0 +1 @@
ass": "state", "iso2": "TZ", "id": "Tanzania"}, {"info": "", "name": "United States", "continent_id": "_North America", "class": "state", "iso2": "US", "id": "USA"}, {"info": "", "name": "Uruguay", "continent_id": "_South America", "class": "state", "iso2": "UY", "id": "Uruguay"}, {"info": "", "name": "Uzbekistan", "continent_id": "_Asia", "class": "state", "iso2": "UZ", "id": "Uzbekistan"}, {"info": "", "name": "Vanuatu", "continent_id": "_Oceania", "class": "state", "iso2": "VU", "id": "Vanuatu"}, {"info": "", "name": "Venezuela", "continent_id": "_South America", "class": "state", "iso2": "VE", "id": "Venezuela"}, {"info": "", "name": "Vietnam", "continent_id": "_Asia", "class": "state", "iso2": "VN", "id": "Vietnam"}, {"info": "", "name": "Yemen", "continent_id": "_Asia", "class": "state", "iso2": "YE", "id": "Yemen"}, {"info": "", "name": "Zambia", "continent_id": "_Africa", "class": "state", "iso2": "ZM", "id": "Zambia"}, {"info": "", "name": "Zimbabwe", "continent_id": "_Africa", "class": "state", "iso2": "ZW", "id": "Zimbabwe"}], "title": "UN Member States:"}, {"states": [{"info": "", "name": "Holy See", "continent_id": "_Europe", "class": "state", "iso2": "VA", "id": "Holy See"}, {"info": "", "name": "Palestine", "continent_id": "_Asia", "class": "state", "iso2": "PS", "id": "Palestine"}], "title": "UN Permanent Observer States:"}, {"states": [{"info": "A fully self-governing autonomous region and was represented as the Republic of China (ROC) at the UN until 1971 when the People's Republic of China (PRC) assumed China's seat.\nCurrently has recognition as a country by 20 UN member states.\n", "name": "Taiwan", "continent_id": "_Asia", "class": "state", "iso2": "TW", "id": "Taiwan"}, {"info": "Declared independence from Serbia in February 2008.\nCurrently has recognition as a country by 115 UN member states as of 27 February 2017.\n", "name": "Kosovo", "continent_id": "_Europe", "class": "state", "iso2": "XK", "id": "Kosovo"}, {"info": "Is formally known as Hong Kong Special Administrative Region of the People's Republic of China. It is a fully autonomous region and maintains a separate political and economic system from China aside from its foreign affairs.\nDue to Hong Kong's long history in the WCA and large community, it is recognized as a separate entity to China within the WCA.\n", "name": "Hong Kong", "continent_id": "_Asia", "class": "state", "iso2": "HK", "id": "Hong Kong"}, {"info": "Is formally known as Macao Special Administrative Region of the People's Republic of China.\nIt is functionally identical to Hong Kong, as so we treat it the same way as Hong Kong within the WCA.", "name": "Macau", "continent_id": "_Asia", "class": "state", "iso2": "MO", "id": "Macau"}], "title": "The following states are non-UN member states recognised by the WCA:"}], "text": "The WCA recognizes a total of 199 independent states. \n\n", "version_hash": "titi", "version": "Version: TODO: Put the date here", "title": "WCA List of Recognized States"}
```

Comment:
Yeah, two more things:
 - I should include the command line executed to generate this file, it should be easier for later reads. Maybe I can just start the json with some kind of "comments" entry.
 - Pretty formatting this will probably make it clearer that there is still a TODO left at the end, I'll actually update this PR with the *real* file once thewca/wca-regulations#442 is ready.